File: smartportfolio/recommendations.py
"""Recommendations module for SmartPortfolio CLI."""

from dataclasses import dataclass
from typing import List

@dataclass
class Recommendation:
    action: str  # "buy" or "sell"
    ticker: str
    reason: str


def list_recommendations(recommendations: List[Recommendation]) -> None:
    """Print buy/sell recommendations."""
    print("\n===== Recommendations =====")
    for rec in recommendations:
        print(f"{rec.action.title()} {rec.ticker} - {rec.reason}")
    print()

File: smartportfolio/recommendations.js
function capitalize(str) {
  return str.charAt(0).toUpperCase() + str.slice(1);
}

class Recommendation {
  constructor(action, ticker, reason) {
    this.action = action;
    this.ticker = ticker;
    this.reason = reason;
  }
}

function listRecommendations(recommendations) {
  console.log('\n===== Recommendations =====');
  recommendations.forEach(rec => {
    console.log(`${capitalize(rec.action)} ${rec.ticker} - ${rec.reason}`);
  });
  console.log();
}

module.exports = { Recommendation, listRecommendations };
